scripts/buildkite/rebuild.hs

```diff
@@ -1,5 +1,6 @@
 {-# LANGUAGE LambdaCase          #-}
```

Comment:
is this file being changed intentionally?

Reply:
reverted this back to the version from master with #9025568